services/hanke-service/src/test/kotlin/fi/hel/haitaton/hanke/ControllerTest.kt

```diff
@@ -17,7 +17,14 @@ interface ControllerTest {
     /** Send a GET request to the given URL. */
     fun get(
         url: String,
-        resultType: MediaType = MediaType.APPLICATION_JSON,
+    ): ResultActions {
```

Comment:
This will stop almost all current controller tests from checking that the Content-Type header is set correctly.

Reply:
Added `getRaw` instead of changing this method.